Error: TAR_BAD_ARCHIVE: Unrecognized archive format

I am trying to install this package and am getting the following:

```shell
➜ npm i --save-dev pmd-bin                                                    
npm ERR! code 1
npm ERR! path /.../node_modules/pmd-bin
npm ERR! command failed
npm ERR! command sh -c node download-java.js
npm ERR! Error: TAR_BAD_ARCHIVE: Unrecognized archive format
```

When pulling down this repo and running `npm i` I am getting the same error.

I was able to resolve it by updating [download-java.js](https://github.com/amtrack/pmd-bin/blob/master/download-java.js) to use a different syntax for installing:

```javascript
.install({ feature_version: 15, type: "jre", allow_system_java: false })
```

It looks like the parameters were changed in a more recent version - and not a major version, so the `^` specified in the package.json version is installing latest.

---

```
system macOS Monterey 12.2.1 (M1)
npm 8.3.1
node v16.14.0
```

--- a/download-java.js
+++ b/download-java.js
@@ -1,7 +1,7 @@
 async function installJava() {
   const jre = require("node-java-connector");
   return jre
-    .install(15, { type: "jre", allow_system_java: false })
+    .install({ feature_version: 17, type: "jre", allow_system_java: false })
     .then((dir) => {
       if (!dir) {
         console.log("using system-wide java installation");

--- a/package.json
+++ b/package.json
@@ -7,7 +7,7 @@
     "pmd": "./pmd"
   },
   "dependencies": {
-    "node-java-connector": "^1.0.1"
+    "node-java-connector": "1.1.1"
   },
   "devDependencies": {
     "adm-zip": "^0.5.9",

